app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -298,6 +298,10 @@ def generate_curl_config
         # determine if this a single-study bulk download (from download tab) or from home page search
         search_context = params[:context] || 'unknown'
 
+        # determine what format we need for output paths based on client OS
+        browser = Browser.new(request.env['HTTP_USER_AGENT'])
+        os = browser&.platform&.name || 'Mac OS X'
```

Comment:
did you mean to include all the windows download code in this PR?  It looks good, but you didn't mention it in your comments